spec.bs

```diff
@@ -219,48 +184,21 @@ Modifications to the HTML Standard {#mod}
 
 ### HTML: <a href="https://html.spec.whatwg.org/#task-source">task source</a> ### {#html-task-source-dfn}
 
-Each task source is classified as <dfn for="task source">freezable</dfn> or <dfn for="task source">unfreezable</dfn>.
+All task sources formally specified (in HTML, CSS, and other specifications) are <dfn for="task source">freezable</dfn>.
 
 All tasks queued on [=task source/freezable=] task source are automatically freezable.
 
-Examples of [=task source/freezable=] task sources are:
+Examples of specified task sources are:
 
 * timers: callbacks generated by timers of a Document
 * networking: tasks queued due to loading of resources
 * DOM manipulation, user interaction, history traversal etc.
 
-<table><tbody>
-<tr><th>Task Queue</th><th>Freezable</th></tr>
-<tr><td>[=DOM manipulation task source=]</td><td>Yes</td></tr>
-<tr><td>[=user interaction task source=]</td><td>Yes</td></tr>
-<tr><td>[=networking task source=]</td><td>Yes* (pending request inflight allowed)</td></tr>
-<tr><td>[=history traversal task source=]</td><td>Yes</td></tr>
-<tr><td>[=embed task source=]</td><td>Yes</td></tr>
-<tr><td>[=media element event task source=]</td><td>Yes</td></tr>
-<tr><td>[=canvas blob serialisation task source=]</td><td>Yes</td></tr>
-<tr><td>[=microtask task source=]</td><td>Yes</td></tr>
-<tr><td>[=timer task source=]</td><td>Yes</td></tr>
-<tr><td>[=remote event task source=]</td><td>Yes</td></tr>
-<tr><td>[=websocket task source=]</td><td>Yes</td></tr>
-<tr><td>[=posted message task source=]</td><td>Yes</td></tr>
-<tr><td>[=unshipped port message queue=]</td><td>Yes</td></tr>
-<tr><td>[=file reading task source=]</td><td>Yes</td></tr>
-<tr><td>[=database access task source=]</td><td>Yes</td></tr>
-<tr><td>[=presentation task source=]</td><td>Yes</td></tr>
-<tr><td>[=sensor task source=]</td><td>Yes</td></tr>
-<tr><td>[=performance timeline task source=]</td><td>Yes</td></tr>
-<tr><td>[=WebGL task source=]</td><td>Yes</td></tr>
-<tr><td>[=idle task task source=]</td><td>Yes</td></tr>
-<tr><td>[=font loading task source=]</td><td>Yes</td></tr>
-<tr><td>[=application life cycle task source=]</td><td>Yes</td></tr>
-<tr><td>[=background fetch task source=]</td><td>Yes</td></tr>
-<tr><td>[=permission task source=]</td><td>Yes</td></tr>
-<tr><td>[=client message queue task source=]</td><td>Yes</td></tr>
-</table>
-
 NOTE: networking task source is freezable with an exception: when a pending fetch keep-alive request in inflight, then networking task source must not be frozen.
 
-Examples of [=task source/unfreezable=] task sources are:
+The UA may have some unspecified task sources that it may wish not to be frozen.
+
+Examples of unspecified task sources are:
```

Comment:
I don't think specs should talk about unspecified things. I think we can just radically simplify the model. All tasks are freezable. So we don't even need the notion of freezeable task sources, or freezable tasks.

The fact that Chrome uses the word "tasks", and shares implementation paths, for some things that are not spec-tasks, should not add complexity to the spec.

So basically we can remove this entire section, and just change https://wicg.github.io/page-lifecycle/spec.html#html-event-loop to ignore frozen documents.

Reply:
Done